builder/azure/arm/builder.go

```diff
@@ -131,6 +132,10 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 	if b.config.ClientConfig.ObjectID == "" && b.config.OSType != constants.Target_Linux {
 		return nil, fmt.Errorf("could not determine the ObjectID for the user, which is required for Windows builds")
 	}
+	publicIPWarning := "On 31 March 2025 Azure is disabling the ability to create public IP Addresses with the Basic SKU, this builder has always created a basic SKU public IP Address to connect to the build VM.  We reccomend setting the `public_ip_sku` field to `Standard`, this will make sure that your build is created with a standard public IPs, and help ensure any issues with this change are caught before the removal of Basic SKU public IPs. At that point this plugin will be updated to default to Standard IP skus. You can read more about this in the official Azure announcement https://azure.microsoft.com/en-us/updates/upgrade-to-standard-sku-public-ip-addresses-in-azure-by-30-september-2025-basic-sku-will-be-retired/"
+	if b.config.PublicIpSKU == "" || b.config.PublicIpSKU == string(publicipaddresses.PublicIPAddressSkuNameBasic) {
```

Comment:
This warning will output for every user once this is released. If the intent it to alert users starting now, we need to update the messaging above to inform the user on how to remove this warning. Which appears to be just using a `Standard` SKU. 

If a user is unable to use a Standard SKU we will keep alerting if they set the field to the only value they can use `Basic`.  I wonder if it makes sense to warn if the value is explicitly set to Basic and in the next two releases update the warning to include those users who are not setting the value.

Reply:
I would rather warn always, and if users can not upgrade now, they will open issues which will reveal the reasons why they can not upgrade, and allow us to fix it, we want users to upgrade as soon as possible so 

1.) We can find out any missing functionality before the deadline
2.) Users are on the new IP sku long before the deadline